### ISSUE TITLE
Use HTTPS by default

### ISSUE THREAD

**Issue by mzur** [fathomnet/community-feedback#28]
Unencrypted web traffic is discouraged nowadays. I suggest that fathomnet.org should use HTTPS by default. Usually, visitors opening an HTTP URL should be redirected to the HTTPS URL.

It seems like the JavaScript isn't working on the HTTPS page, though. Probably it tries to access HTTP URLs which isn't allowed by the browser.

**Comment by hohonuuli** [comment]
@mzur Yes, I totally agree. We have the the SSL layer configured and ready to roll. However, we still have to work though some issues with required HTTP headers from the backend servers getting clobbered by the proxy server when SSL is used. Once the workshop is done, we'll bump the priority of this task up.

**Comment by hohonuuli** [comment]
Closing this issue as it's a duplicate of #12

**Comment by hohonuuli** [comment]
Follow up note ... all FathomNet web traffic is now redirected to HTTPS.
